src/main/java/cart/dto/CartItemQuantityUpdateRequest.java

```diff
@@ -1,6 +1,12 @@
 package cart.dto;
 
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
 public class CartItemQuantityUpdateRequest {
+
+    @NotNull(message = "수량을 입력해야 합니다.")
+    @Min(value = 0, message = "수량은 음수가 될 수 없습니다.")
```

Comment:
C : ${validatedValue}를 이용해서 사용자가 입력한 값을 알려줄 수 있어!
```suggestion
    @NotNull(message = "수량을 입력해야 합니다. 입력값 : ${validatedValue}")
    @Min(value = 0, message = "수량은 음수가 될 수 없습니다. 입력값 : ${validatedValue}")
```

Reply:
고맙습니당 반영해뒀어!